packages/sx05re/emuelec/bin/setres.sh

```diff
@@ -35,53 +25,45 @@ blank_buffer()
 }
 
 BPP=32
-HZ=60
-
 MODE=$1
-CUR_MODE=`cat /sys/class/display/mode`;
 
 # If the current display is the same as the change just exit.
 [ -z "$MODE" ] && exit 0;
 [[ $MODE == "auto" ]] && exit 0;
-# Removed because if try to set invalid video mode it becomes a valid MODE
-# in display/mode and then when trying to revert back this becomes true exiting.
-#[[ "$MODE" == "$CUR_MODE" ]] && exit 0;
 
 if [[ ! "$MODE" == *"x"* ]]; then
   case $MODE in
-  	*p*) H=$(echo $MODE | cut -d'p' -f 1) ;;
-  	*i*) H=$(echo $MODE | cut -d'i' -f 1) ;;
-  	*cvbs*) H=$(echo $MODE | cut -d'c' -f 1) ;;
+    *p*) H=$(echo $MODE | cut -f'p' -f 1) ;;
+    *i*) H=$(echo $MODE | cut -f'i' -f 1) ;;
   esac
 fi
 
-HZ=${MODE:(-4):2}
-if [[ ! -n "$HZ" ]] || [[ $HZ -eq 50 ]]; then
-	HZ=60
-fi
-
 
 # hides buffer
 show_buffer 1
 
+if [[ "$EE_PROJECT" == "Rockchip" ]]; then
```

Comment:
Where is this variable `EE_PROJECT` coming from?

Reply:
It's not a variable yet, but it can be set in the profile script if there is Rockchip devices with hdmi output.

Reply:
project rockchip change removed.